ZukoAzula2016/src/org/team1540/zukoazula/ShooterSpeedControl.java

```diff
@@ -0,0 +1,45 @@
+package org.team1540.zukoazula;
+
+import ccre.channel.BooleanInput;
+import ccre.channel.EventCell;
+import ccre.channel.EventInput;
+import ccre.channel.FloatCell;
+import ccre.channel.FloatInput;
+import ccre.channel.FloatOutput;
+import ccre.cluck.Cluck;
+import ccre.ctrl.ExtendedMotor;
+import ccre.ctrl.ExtendedMotorFailureException;
+import ccre.frc.FRC;
+import ccre.time.Time;
+
+public class ShooterSpeedControl {
+    public static final ExtendedMotor shooterCAN = FRC.talonCAN(7);
+    public static BooleanInput shooting;
+    public static FloatOutput shooterMotor;
+    public static FloatInput gearTooth;
```

Comment:
Should I keep the motor public so other classes can use it for emergency shutoff or whatever?